src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/UnusedImportsCheck.java

```diff
@@ -231,14 +231,26 @@ private boolean isUnusedImport(String imprt) {
     private void processIdent(DetailAST ast) {
         final DetailAST parent = ast.getParent();
         final int parentType = parent.getType();
-        if (parentType != TokenTypes.DOT
+        if ((parentType != TokenTypes.DOT
```

Comment:
looks line additional paren is redundant

Reply:
Removing the parentheses will give a false positive because of the order of precedance.

Reply:
Sorry, I missed that closing brace is not at the end of expression